### ISSUE TITLE
Weird loss graph and poor results

### ISSUE THREAD

**Issue by cdsuperstar** [zylo117/Yet-Another-EfficientDet-Pytorch#493]
![image](https://user-images.githubusercontent.com/13405392/91373380-d4944e80-e848-11ea-93ec-87260b7949c1.png)
Help me check the loss graph please, it's means lr larger or something else?
After 37 epoch, there are poor results like the this:
![image](https://user-images.githubusercontent.com/13405392/91373604-69974780-e849-11ea-8241-40da1cd54344.png)
Training datasets have 4 classes,  class A:1304 images, class B:14 images, class C:360 images, class D:784 images.
They are all grayscale images, any idea help improve the results?

**Comment by whwxyshpy** [comment]
the problem is the same as me ,so weired

**Comment by LoveIsAGame** [comment]
我也一样，请问你的这个问题解决了吗？

**Comment by codefabulous** [comment]
Hi, how do you generate the loss graph?
